core/perf_test/CMakeLists.txt

```diff
@@ -114,3 +114,91 @@ IF(NOT Kokkos_ENABLE_OPENMPTARGET)
     CATEGORIES PERFORMANCE
   )
 ENDIF()
+
+
+IF(NOT Kokkos_ENABLE_BENCHMARKS OR KOKKOS_HAS_TRILINOS)
+  RETURN()
+ENDIF()
+
+find_package(benchmark QUIET)
+IF(NOT benchmark_FOUND)
+  message(STATUS "No installed benchmark found, fetching from GitHub")
```

Comment:
I would say 'google benchmark' instead of 'benchmark'. Otherwise it could be read as we are downloading the benchmarks

Reply:
Done (7f10e39799487c7ed30157a27a94764914960b8a).